Dockerfile

```
@@ -25,9 +24,14 @@ FROM prepare AS build
 ENV CEDAR_SPEC_ROOT=/opt/src/cedar-spec
 COPY . $CEDAR_SPEC_ROOT
 
-# Build DRT
+# Clone `cedar` repository
 WORKDIR $CEDAR_SPEC_ROOT
-RUN source /root/.profile && ./build.sh
+RUN git clone https://github.com/cedar-policy/cedar
```

Minor improvement to image build time and size
```suggestion
RUN git clone --depth 1 https://github.com/cedar-policy/cedar
```